set.mm

```diff
@@ -93573,11 +93606,64 @@ singletons being the empty set ( ` A e/ _V ` ).  (Contributed by AV,
   $}
 
   ${
-    $d a f x y A $.
+    $d A x y $.
+    $( A set that has at least 2 different members dominates ordinal 2.
+       (Contributed by BTernaryTau, 30-Dec-2024.) $)
+    rex2dom $p |- ( ( A e. V /\ E. x e. A E. y e. A x =/= y ) -> 2o ~<_ A ) $=
+      ( wcel cv wne wrex c2o cdom wbr cvv wi cpr cen c0 c1o a1i vex syl elex wa
+      wss prssi df2o3 0ex 1oex 1n0 necomi en2prd eqbrtrid domssr 3expib syl2ani
+      id endom expd rexlimdvv imp ) CDEZAFZBFZGZBCHACHZICJKZUTCLEZVDVEMCDUAVFVC
+      VEABCCVFVACEVBCEUBZVCVEVGVFVAVBNZCUCZIVHJKZVEVCVAVBCUDVCIVHOKVJVCIPQNVHOU
+      EVCPQVAVBLLLLPLEVCUFRQLEVCUGRVALEVCASRVBLEVCBSRPQGVCQPUHUIRVCUOUJUKIVHUPT
+      VFVIVJVEIVHCLULUMUNUQURTUS $.
+    $( $j usage 'rex2dom' avoids 'ax-pow' 'ax-un'; $)
+  $}
+
+  ${
+    $d A x y $.  $d A f x $.
+    $( Strict dominance over 1 is the same as dominance over 2.  (Contributed
+       by BTernaryTau, 23-Dec-2024.) $)
+    1sdom2dom $p |- ( 1o ~< A <-> 2o ~<_ A ) $=
+      ( vx vy vf c1o csdm wbr c2o cdom cvv wcel cv wne csn wceq wex c0 syl wtru
+      cen wrex relsdom brrelex2i wo sdomdom 0sdom1dom sylibr wb mpbid n0snor2el
+      0sdomg sdomnen df1o2 0ex vex en2sn mp2an eqbrtri breq2 exlimiv nsyl olcnd
+      mpbiri rex2dom syl2anc wn wss cpr snsspr1 3sstr4i domssl mpan wf1 snnen2o
+      df2o3 a1i 1oex 1n0 nesymi enpr2d mptru breq1 mpbii mto 2on0 f1cdmsn mpan2
+      nex brdomi mtbiri con2i nexdv wf1o reldom breng wi ccnv f1of1 f1eq3 ax-mp
+      f1ocnv sylib sylan expcom exlimdv sylbid mtod brsdom sylanbrc impbii ) EA
+      FGZHAIGZXKAJKZBLZCLZMCAUABAUAZXLEAFUBUCZXKXPAXNNZOZBPZXKAQMZXPXTUDXKQAFGZ
+      YAXKEAIGZYBEAUEAUFZUGXKXMYBYAUHZXQAJUKZRUIBCBAUJRXKEATGZXTEAULXSYGBXSYGEX
+      RTGEQNZXRTUMQJKZXNJKYHXRTGUNBUOQXNJJUPUQURAXRETUSVCUTVAVBBCAJVDVEXLYCYGVF
+      XKEHVGXLYCYHQEVHZEHQEVIUMVOVJEHAVKVLZXLYGXTXLXSBXSXLXSXLHXRIGZYLHXRDLZVMZ
+      DPYNDYNHXONZOZCPZYPCYPYOHTGZXOVNYPHHTGYRHYJHTVOYJHTGSQEJJYISUNVPEJKZSVQVP
+      QEOVFSEQVRVSVPVTWAURHYOHTWBWCWDWHYNHQMYQWECHXNYMWFWGWDWHHXRDWIWDAXRHIUSWJ
+      WKWLXLYGEAYMWMZDPZXTXLXMYGUUAUHZHAIWNUCZYSXMUUBVQEADJJWOVLRXLYAUUAXTWPXLY
+      BYAXLYCYBYKYDUGXLXMYEUUCYFRUIYAYTXTDYTYAXTYTAYHYMWQZVMZYAXTYTAEUUDWMZUUEE
+      AYMXAUUFAEUUDVMZUUEAEUUDWREYHOUUGUUEUHUMEYHAUUDWSWTXBRBAQUUDWFXCXDXERXFXG
+      EAXHXIXJ $.
+    $( $j usage '1sdom2dom' avoids 'ax-pow' 'ax-un'; $)
+  $}
+
+  ${
+    $d A x y $.
     $( A set that strictly dominates ordinal 1 has at least 2 different
        members.  (Closely related to ~ 2dom .)  (Contributed by Mario Carneiro,
-       12-Jan-2013.) $)
+       12-Jan-2013.)  Avoid ~ ax-pow , ~ ax-un .  (Revised by BTernaryTau,
```

Comment:
[1sdom](https://us.metamath.org/mpeuni/1sdom.html) does not depend on ax-pow, so the revision comment is incorrect.  This avoids ax-un only.

Reply:
Fixed.